samples/nrf9160/mqtt_simple/README.rst

```diff
@@ -9,6 +9,9 @@ nRF9160: Simple MQTT
 
 The Simple MQTT sample demonstrates how to easily connect an nRF9160 SiP to an MQTT broker and send and receive data.
 
+.. note::
+   This sample has been deprecated. To test MQTT connectivity use the the new :ref:`mqtt_sample` sample.
```

Comment:
I'm not sure I follow, what's the point of deprecating a sample? Can't it just be removed?

Reply:
IMO release notes are the place to mention that.

Reply:
Didn't know whether to keep it or not. I certainly don't have any objection on removing it if that is the popular opinion.
@wbober Do you mean the new samples rst or the old one? Is it perhaps sufficient enough to mention the removal in the changelog.

Reply:
OK. Let's go with changelog/release notes.